Gemfile.lock

```diff
@@ -244,17 +245,24 @@ GEM
     net-smtp (0.5.0)
       net-protocol
     nio4r (2.7.4)
-    nokogiri (1.16.7-aarch64-linux)
+    nokogiri (1.18.7)
```

Comment:
Not sure why nokogiri is being upgraded here, might be to do with the local config.